GMX-Bots/Large-Open-and-Increase-Positions/package.json

```diff
@@ -21,20 +20,22 @@
     "disable": "forta-agent disable",
     "enable": "forta-agent enable",
     "keyfile": "forta-agent keyfile",
-    "test": "jest --detectOpenHandles",
-    "format": "prettier --write \"src/**/*.ts\""
+    "format": "prettier --write \"src/**/*.ts\"",
+    "test": "jest --detectOpenHandles"
   },
   "dependencies": {
     "forta-agent": "^0.1.6"
   },
   "devDependencies": {
     "@types/jest": "^27.0.1",
     "@types/nodemon": "^1.19.0",
-    "forta-agent-tools": "^2.0.6",
+    "bignumber.js": "^9.0.2",
+    "forta-agent-tools": "^2.0.7",
     "jest": "^27.0.6",
     "nodemon": "^2.0.8",
-    "prettier": "^2.6.0",
+    "prettier": "^2.6.2",
     "ts-jest": "^27.0.3",
     "typescript": "^4.3.4"
-  }
+  }, 
+  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/GMX-Bots/Large-Open-and-Increase-Positions"
```

Comment:
repo doesn't match the name of the folder

Reply:
the `repository` field actually matches the folder name. appreciate your review :)